src/coreclr/jit/emitxarch.cpp

```diff
@@ -12752,13 +12752,13 @@ BYTE* emitter::emitOutputCV(BYTE* dst, instrDesc* id, code_t code, CnsVal* addc)
         dst += emitOutputByte(dst, 0x64);
     }
 
-    // Compute VEX prefix
-    // Some of its callers already add VEX prefix and then call this routine.
-    // Therefore add VEX prefix is not already present.
-    code = AddVexPrefixIfNeededAndNotPresent(ins, code, size);
+    // Compute VEX/VEX prefix
```

Comment:
`EVEX/VEX` ?

Reply:
yes! Will fix it in the final  non draft version. Thanks for catching that

Reply:
Done!